src/utils/tokens.ts

```diff
@@ -86,12 +86,21 @@ export const getTokenMap = async (
     const { address } = tokenInfo;
     try {
       new PublicKey(address);
-      tokenMap[address] = overrideCoingeckoId(tokenInfo);
+      const ti = tokenMap[address];
```

Comment:
An alternative to this might be to evaluate the Saber token list before the Solana token list, and prefer Solana's `token-list` values. Saber is pretty good at keeping `token-list` up-to-date with their assets